schema/component.schema.json

```diff
@@ -2,7 +2,7 @@
   "$anchor": "confidenceSlider-component",
   "$schema": "https://json-schema.org/draft/2020-12/schema",
   "type": "object",
-  "$patch": {
+  "$merge": {
```

Comment:
What's the requirement to change this from `$patch` to `$merge` or even, what does it do?

Reply:
They are meant to have different `with` formats when using them from their library [ajv-merge-patch](https://www.npmjs.com/package/ajv-merge-patch).
AATV1 Seems to treat them both as `$merge` style, as it implements it's own version of [`$merge and $patch`](https://github.com/adapt-security/adapt-authoring-jsonschema/blob/cd650d6bcff85fdb1b773f1b6fdee0b59333aff1/lib/JsonSchema.js#L170-L195
) and doesn't use the library. 

Ref: 
https://www.npmjs.com/package/ajv-merge-patch
https://ajv.js.org/guide/combining-schemas.html#merge-and-patch-keywords

Reply:
Thank you for the explanation